src/main/java/org/broadinstitute/consent/http/resources/DataAccessRequestResource.java

```diff
@@ -182,11 +182,33 @@ public Response getInvalidDataAccessRequest() {
     @Path("/{id}")
     @Produces("application/json")
     @PermitAll
-    public Document describe(@PathParam("id") String id) {
-        return dataAccessRequestAPI.describeDataAccessRequestById(id);
+    public Response describe(@PathParam("id") String id) {
+        try {
+            new ObjectId(id);
+        } catch (IllegalArgumentException e) {
+            String message = "The provided id is not a valid Data Access Request Id: " + id;
+            logger.log(Level.INFO, message + "; " + e.getMessage());
+            return Response
```

Comment:
```suggestion
            return createExceptionResponse(new IllegalArgumentException(message));
```
Maybe it could be simplified like this